core/src/main/java/org/keycloak/representations/account/ClientRepresentation.java

```diff
@@ -22,4 +28,52 @@ public String getClientName() {
     public void setClientName(String clientName) {
         this.clientName = clientName;
     }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isUserConsentRequired() {
+        return userConsentRequired;
+    }
+
+    public void setUserConsentRequired(boolean userConsentRequired) {
+        this.userConsentRequired = userConsentRequired;
+    }
```

Comment:
Should we rename this to "internal"? I know this could be done on the front end, but I'd say let's do it here – let backend be responsible for this.

Reply:
@vmuzikar Stian asked for this to be renamed from "internal" to "userConsentRequired" in the last round of reviews.

Reply:
If I got it right, @stianst wanted to set "internal" based on "consentRequired", not to rename it to it.

Reply:
Stian said:
What is "internal"? Is it true for clients that don't require consent, and false for clients that do require consents? I don't think that's clear and could be easily mistaken. I would suggest userConsentRequired: true|false